core/src/provider/rpc.rs

```diff
@@ -230,13 +230,12 @@ impl BlockDataProvider for RpcBlockDataProvider {
 
         let batch_limit = 1000;
         while !accounts.is_empty() {
-            if cfg!(debug_assertions) {
-                raiko_lib::inplace_print(&format!(
-                    "fetching storage proof {idx}/{num_storage_proofs}..."
-                ));
-            } else {
-                trace!("Fetching storage proof {idx}/{num_storage_proofs}...");
-            }
+            #[cfg(debug_assertions)]
```

Comment:
what's the real difference??

Reply:
`if cfg!(..)` checks and makes decision at runtime, and `#[cfg(..)]` at compile-time. 

`#[cfg(..)]` is more efficient in terms of binary size and performance:
- binary size: only one code branch will be included
- performance: no runtime overhead since the check is done at compile-time